google/ggrc-core
# GGRC-5912,GGRC-5919 Make Assertions for Controls Mandatory Field

# Issue description

The first case:
Make Assertions for controls mandatory field, make Assertions for controls mandatory during import. Also import template for controls should contain snowflake like for other mandatory fields.

The second case:
Assertions/Categories are not displayed on Control Info panel after creating a new control.

# Steps to test the changes

The first case:

Try to import control with blank assertions field.

**Expected result:** "Missing mandatory column Assertions, when adding object." error is shown.

The second case:

1. Log in to GGRC and create a Control with Assertions and Categories 
2. Look at the Assertions and Categories on Control info panel: are missed

# Solution description

Added mandatory flag in _aliases for AssertionCategorized

**FE part:**
Add non blank validation of controls assertions
Set default value for "selected" (when you create new control, instance does not have assertions so it is undefined)
Add default values for `multiselect-dropdown-wrapper`
Fix styles for assertions on modal
Add asterisk label on info-pane

# Sanity checklist




# PR Review checklist
